crates/net/network/src/transactions.rs

```diff
@@ -1204,6 +1222,12 @@ enum TransactionsCommand {
         peers: Vec<PeerId>,
         tx: oneshot::Sender<HashMap<PeerId, HashSet<TxHash>>>,
     },
+    /// Request specific transactions from a peer.
+    TransactionsHandle {
+        peer_id: PeerId,
+        requested_hashes: Vec<TxHash>,
+        result: oneshot::Sender<Result<RequestResult<PooledTransactions>, RecvError>>,
```

Comment:
it's easier to return a clone of the Sender if the peer exists
```suggestion
        result: oneshot::Sender<Option<PeerRequestSender>>,
```